src/components/link/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react'
+import classNames from 'classnames'
+import styles from './styles.module.css'
+
+interface Props extends React.HTMLAttributes<HTMLAnchorElement> {
+  children: ReactNode
+  className?: string
+  href: string
+}
+
+const Link = ({ children, className, href, ...rest }: Props) => (
+  <a href={href} className={classNames(styles.link, className, 'align-center p-2')} {...rest}>
+    {children}
+  </a>
+)
```

Comment:
```suggestion
const Link = ({ className, ...rest }: React.HTMLAttributes<HTMLAnchorElement>) => (
  <a className={classNames(styles.link, 'align-center p-2', className)} {...rest} />
)
```

Reply:
Type '{ children: Element; title: string; href: string; }' is not assignable to type 'IntrinsicAttributes & HTMLAttributes<HTMLAnchorElement>'.
  Property 'href' does not exist on type 'IntrinsicAttributes & HTMLAttributes<HTMLAnchorElement>'.

Obligé de déclarer le href en Props

Reply:
ah mais oui j'ai pas fait attention... c'est pas un a qu'il faut que tu mettes, mais un Link de next, avec du coup les LinkProps :)